app/components/spina/forms/search_component.html.erb

```diff
@@ -1,16 +1,12 @@
 <div class="flex items-center relative">
-  <div class="absolute left-3" style="left: 0.75rem;">
+  <div class="absolute left-2">
     <%= helpers.heroicon('search', style: :solid, class: 'text-gray-300 w-5 h-5') %>
   </div>
 
-  <%= @f.search_field @method, 
+  <%= f.search_field @method, 
     value: params[@method.to_sym],
-    class: "form-input font-medium block pl-10 pr-2 py-[0.3rem]"
+    placeholder: t("spina.search"),
+    class: "form-input rounded-full pl-8 text-sm h-9 w-full",
+    data: {action: "input->form#requestSubmit"}
   %>
-
-  <div class="absolute right-2">
-    <%= link_to request.path do %>
-      <%= helpers.heroicon('x', style: :solid, class: 'text-gray-300 w-5 h-5') %>
```

Comment:
My initial development was in Firefox, and I didn't consider that Safari added its own close button. It makes sense to leave the management of the field to the browser.